res/layout/uploader_layout.xml

```diff
@@ -43,33 +36,75 @@
 			android:dividerHeight="1dip">
 		</ListView>
 
+		<TextView android:layout_width="fill_parent"
+            android:text="@string/uploader_top_message"
+            android:layout_height="wrap_content"
+            android:id="@+id/drawer_username"
+            android:textColor="@android:color/black"
```

Comment:
Isn't there a color/owncloudDarkTextColor (or something similar) resource in the project?